lib/compilers/typescript-native.ts

```diff
@@ -45,6 +46,7 @@ export class TypeScriptNativeCompiler extends BaseCompiler {
 
         this.tscJit = this.compilerProps<string>(`compiler.${this.compiler.id}.exe`);
         this.tscSharedLib = this.compilerProps<string>(`compiler.${this.compiler.id}.sharedlibs`);
+        this.tscVersion = /tsc-\d+\.\d+-pre-alpha(\d+)/g.exec(this.tscJit)?.[1] || "0";
```

Comment:
This seems pretty fragile in general: relying on the name of the executable to contain the version. I'm ok enough with it for this fringe case though.

I am pretty sure that `this.tscJit` is the same as `this.compiler.exe`, while we're here

Reply:
updated code